### DISCUSSION TITLE
【2411910】嘣境回收战

### DISCUSSION THREAD

**Discussion by hisoio** [pjy612/SteamManifestCache#145] [Share]
appid:2411910
keys
```
2411911;ea3a207f668630bd56f125c6aeffa31c503531ce201f0bd27cbf99bbf77cc163
```
CODE:34E5B8D0ACEF40D1
E-mail：905633661@qq.com
大佬，求个内测~

**Comment by pjy612** [comment]
未收录 烦请附带清单

**Reply by hisoio** [reply]
[清单.zip](https://github.com/pjy612/SteamManifestCache/files/13761237/default.zip)
大佬 清单

**Reply by pjy612** [reply]
感谢 查收

**Reply by hisoio** [reply]
给你发邮箱了大佬

**Reply by pjy612** [reply]
emm 没收到... 不过 我找到了

**Reply by hisoio** [reply]
大佬 内测码变了 不知道怎么入库不成功 游戏没显示 code:91DF6FFEF68F20B9 能重新弄下么

**Reply by hisoio** [reply]
以前的游戏入库的 现在不能玩显示购买 重新入库也不行 不知道怎么回事

**Reply by pjy612** [reply]
看release 更新记录 重新入库
